### PR TITLE
fix: stabilize april tags by adding a temporal filter to april tag de…

### PR DESCRIPTION
**Pull request by DrCodeWizzard**
…tector

### CODE REVIEW

File: apriltags_ros/src/apriltag_detector.cpp [hunk 1]
```diff
@@ -99,6 +106,17 @@ AprilTagDetector::AprilTagDetector(ros::NodeHandle& nh, ros::NodeHandle& pnh) :
   pnh.param<float>("plane_angle_threshold", plane_angle_threshold_,0.0872665f);
   plane_angle_threshold_ = std::max(0.0f, std::min(90.0f, plane_angle_threshold_));
 
+
+  pnh.param<float>("tf_pose_acceptance_error_range", tf_pose_acceptance_error_range_, 4);
+  tf_pose_acceptance_error_range_ = std::max(0.0f, std::min(90.0f, tf_pose_acceptance_error_range_));
```
Comment (grieneis) [inline apriltags_ros/src/apriltag_detector.cpp]: Our other angle is in radians, and this is in degrees.  I'd prefer to keep everything in rad.

File: apriltags_ros/src/apriltag_detector.cpp [hunk 2]
```diff
@@ -99,6 +106,17 @@ AprilTagDetector::AprilTagDetector(ros::NodeHandle& nh, ros::NodeHandle& pnh) :
   pnh.param<float>("plane_angle_threshold", plane_angle_threshold_,0.0872665f);
   plane_angle_threshold_ = std::max(0.0f, std::min(90.0f, plane_angle_threshold_));
 
+
+  pnh.param<float>("tf_pose_acceptance_error_range in radians", tf_pose_acceptance_error_range_, 4);
```
Comment (zachfang) [inline apriltags_ros/src/apriltag_detector.cpp]: Is this a typo? Looks like underscore are missing

Comment (grieneis) [reply apriltags_ros/src/apriltag_detector.cpp]: If it is in radians, the default should not be 4.

Comment (DrCodeWizzard) [reply apriltags_ros/src/apriltag_detector.cpp]: yes fixed typos

**Comment by zachfang** [comment]
Found some extra blank lines in the code change. For example: https://github.com/6RiverSystems/apriltags_ros/pull/17/files#diff-e394dcd963050060a43511814f2ee917R390

Other than that, looks good to me

**Review by zachfang — APPROVED** [review]

**Pull request merged**